pkg/ovs/openflow/interfaces.go

```diff
@@ -68,6 +68,11 @@ type Bridge interface {
 	GetName() string
 	DeleteTable(id TableIDType) bool
 	DumpTableStatus() []TableStatus
+	// DumpFlows queries the Openflow entries from OFSwitch, the filter of the query is Openflow cookieID. The result is
```

Comment:
, the filter -> . The filter

Maybe also change: cookieID; the result ...

Reply:
Done.